Sources/Sentry/SentryFramesTracker.m

```diff
@@ -166,7 +166,9 @@ - (void)unpause
     }
 
     _isRunning = YES;
-
+    // Reset the previous frame timestamp to avoid false app hang detection
+    // This also means we will not detect a hang if the app stuck while the tracker was paused.
```

Comment:
`m`: The frames tracker doesn't know about app hangs. This here is solely about wrong frame statistics, which has the positive side effect of getting rid of wrong app hangs as well, because app hangs v2 uses the frame statistics.